scripts/style.sh

```diff
@@ -25,6 +25,7 @@
 #   clang-format version 7.0.0 (tags/google/stable/2018-01-11)
 #   clang-format version google3-trunk (trunk r333779)
 version=$(clang-format --version)
+echo "Found: $version"
```

Comment:
I have a very slight preference that our tools not be noisy during interactive use, but can't really argue with the utility of this in the travis log.

Would you be willing to wrap this in an

```bash
if [[ ! -t 1 ]]; then
  echo ...
fi
```
That way it will show up in log files but not during interactive use.

Reply:
Sure.  Thanks for giving me code to copy/paste.